framework_tvb/tvb/adapters/exporters/export_manager.py

```diff
@@ -235,6 +239,18 @@ def export_project(self, project, optimize_size=False):
 
         return result_path
 
+    @staticmethod
+    def _get_op_with_errors(project_id):
+        """
+        Get the operation folders with error base name as list.
+        """
+        operations = dao.get_operations_in_project(project_id)
+        op_with_errors = []
+        for op in operations:
+            if op.status == STATUS_ERROR:
```

Comment:
The query could return only the operations with error status, so we would not need this check anymore